crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,19 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    platform, version = @node["platform"], @node["platform_version"]
+    case service_name
+    when :chef
+      service_cmd = ChefObject.service_command(platform, version, "chef-client", action)
+    end
+    if service_cmd.nil?
+      Rails.logger.error("service: init command for #{platform} #{version} could not be determined")
```

Comment:
nitpick: the message is about an `init command` but everything else we do is about a `service command` :-)